docs/commands.md

```diff
@@ -7,116 +7,116 @@ Creates a new Function project at _`path`_. If _`path`_ is unspecified, assumes
 ```
 
 ## `list`
 
-Lists all deployed functions. The namespace defaults to the value in `faas.yaml` or the namespace currently active in the user's Kubernetes configuration. The namespace defaults to the value in `faas.yaml` or the namespace currently active in the user's Kubernetes configuration. The namespace may be specified on the command line, and if so this will overwrite the value in `faas.yaml`.
+Lists all deployed functions. The namespace defaults to the value in `function.yaml` or the namespace currently active in the user's Kubernetes configuration. The namespace defaults to the value in `function.yaml` or the namespace currently active in the user's Kubernetes configuration. The namespace may be specified on the command line, and if so this will overwrite the value in `function.yaml`.
 
-Similar `kn` command: `kn service list [name] [flags]`. This command lists all deployed Knative `Services`. As with other `kn` commands that have similar functionality, there is more information and flexibilty in the `kn` command. However, `kn` will return _all_ `Services`, while `faas list` will only display the boson Functions that have been deployed. Consider improving the output of the `faas list` command so that it is at least as informative as `kn service list`.
+Similar `kn` command: `kn service list [name] [flags]`. This command lists all deployed Knative `Services`. As with other `kn` commands that have similar functionality, there is more information and flexibilty in the `kn` command. However, `kn` will return _all_ `Services`, while `function list` will only display the boson Functions that have been deployed. Consider improving the output of the `function list` command so that it is at least as informative as `kn service list`.
 
 ```console
-faas list [-n <namespace> -p <path>]
+function list [-n <namespace> -p <path>]
 ```
 
 When run as a `kn` plugin.
 
 ```console
-kn faas list [-n <namespace> -p <path>]
+kn function list [-n <namespace> -p <path>]
 ```
 
 ## `delete`
 
-Removes a deployed function from the cluster. The user may specify a function by name, path or if neither of those are provided, the current directory will be searched for a `faas.yaml` configuration file to determine the function to be removed. The namespace defaults to the value in `faas.yaml` or the namespace currently active in the user's Kubernetes configuration. The namespace may be specified on the command line, and if so this will overwrite the value in `faas.yaml`.
+Removes a deployed function from the cluster. The user may specify a function by name, path or if neither of those are provided, the current directory will be searched for a `function.yaml` configuration file to determine the function to be removed. The namespace defaults to the value in `function.yaml` or the namespace currently active in the user's Kubernetes configuration. The namespace may be specified on the command line, and if so this will overwrite the value in `function.yaml`.
```

Comment:
function.yaml -> func.yaml (I know this file is generated, but you got the idea ;-).